packages/components/organisms/f-header/src/components/_tests/Logo.test.js

```diff
@@ -44,6 +44,31 @@ describe('Logo', () => {
         expect(logo).toBeDefined();
     });
 
+    it('should change the logo link to a span if linkDisabled prop is provided', () => {
+        const $style = {
+            disabled: 'disabled'
+        };
+        // Arrange
+        const propsData = {
+            theme: 'je',
+            companyName: 'Just Eat',
+            isLogoDisabled: true
+        };
+
+        // Act
+        const wrapper = shallowMount(Logo, {
+            propsData,
+            mocks: {
+                $style
+            }
+        });
+
+        const logo = wrapper.find('span');
```

Comment:
Might be best to name this `logoWrapper` / `logoWrapElement` or something that indicates that it's the container we're testing for. The expect below made me read this setup code as i thought we were testing the logo existence itself.